doc/deoplete.txt

```diff
@@ -581,6 +583,18 @@ dictionary					*deoplete-source-dictionary*
 		This source collects |g:deoplete#keyword_patterns| keywords
 		from 'dictionary'.
 
+		" Examples:
+		" Sample configuration for dictionary source with multiple
+		" dictionary files.
+		setlocal dictionary+=/usr/share/dict/words
+		setlocal dictionary+=/usr/share/dict/american-english
```

Comment:
dictionary source only checks buffer local dictionary.
But set change the global and buffer local value.
It may be works.  But it is not intended.